backup_and_restore/application_backup_and_restore/installing/data-mover-intro.adoc

```diff
@@ -0,0 +1,36 @@
+:_content-type: CONCEPT
+[id="oadp-data-mover-intro"]
+= OADP Data Mover Introduction
+include::_attributes/common-attributes.adoc[]
+:context: data-mover
+
+toc::[]
+
+OADP Data Mover allows you to restore stateful applications from the store if a failure, accidental deletion, or corruption of the cluster occurs.
+
+[NOTE]
+====
+The OADP 1.1 Data Mover is a Technology Preview feature.
+
+The OADP 1.2 Data Mover has significantly improved features and performances, but is still a Technology Preview feature.
+====
+:FeatureName: The OADP Data Mover
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+* You can use OADP Data Mover to back up Container Storage Interface (CSI) volume snapshots to a remote object store. See xref:../../../backup_and_restore/application_backup_and_restore/installing/oadp-using-data-mover-for-csi-snapshots-doc.adoc#oadp-using-data-mover-for-csi-snapshots-doc[Using Data Mover for CSI snapshots].
+
+* You can use OADP 1.2 Data Mover to backup and restore application data for clusters that use CephFS, CephRBD, or both. See xref:../../../backup_and_restore/application_backup_and_restore/installing/oadp-using-data-mover-for-csi-snapshots-doc.adoc#oadp-using-data-mover-for-csi-snapshots-doc[Using OADP 1.2 Data Mover with Ceph storage].
+
+* You must perform a data cleanup after you perform a backup, if you are using OADP 1.1 Data Mover. See xref:../../../backup_and_restore/application_backup_and_restore/installing/oadp-cleaning-up-after-data-mover-1-1-backup-doc.adoc#oadp-cleaning-up-after-data-mover-1-1-backup-doc[Cleaning up after a backup using OADP 1.1 Data Mover].
+
+
+[id="oadp-data-mover-prerequisites"]
+== OADP Data Mover prerequisites
+
+* You have a stateful application running in a separate namespace.
```

Comment:
Let's simplify `You have a stateful application running in a separate namespace.`
to:
`You have an application running in a separate namespace.`